tools/mmp/driver.cs

```diff
@@ -362,7 +372,28 @@ static void Main2 (string [] args)
 				{ "xamarin-framework-directory=", "The framework directory", v => { xm_framework_dir = v; }, true },
 				{ "xamarin-full-framework", "Used with --target-framework=4.5 to select XM 4.5 Target Framework", v => { IsUnifiedFullXamMacFramework = true; } },
 				{ "xamarin-system-framework", "Used with --target-framework=4.5 to select XM 4.5 Target Framework", v => { IsUnifiedFullSystemFramework = true; } },
-				{ "aot", "Enable experimental AOT Xamarin.Mac support", v => { aot = true; } },
+				{ "aot:", "Specify assemblies that should be compiled via experimental AOT.\n- none - No AOT (default)\n- all - Every assembly in MonoBundle not ignored\n- platform - Just Xamarin.Mac.dll, System.dll, and mscorlib.dll\n- explicit - Just the assemblies you pass in to --aot-reference)",
+					v => {
+						switch (v) {
+						case "none":
+							aot = AotType.None;
+							break;
+						case "all":
+							aot = AotType.All;
+							break;
+						case "platform":
```

Comment:
@rolfbjarne Lining up with how mtouch does things is good enough for me. :) I'll rework